scripts/covid_hosp_explore.R

```diff
@@ -1,179 +1,125 @@
-# Run the R scripts in the R/ folder with your custom functions:
-# tar_source()
-# where the forecasters and parameters are joined; see either the variable param_grid or `tar_read(forecasters)`
 source("scripts/targets-common.R")
 source("scripts/targets-exploration-common.R")
 
-#' A simple faux-forecaster handy for debugging pipelines.
-dummy_forecaster <- function(...) {
-  dots <- list(...)
-  tibble(
-    forecaster = "dummy",
-    geo_value = "d",
-    forecast_date = as.Date("2024-01-01"),
-    target_end_date = forecast_date + pluck(dots, "ahead", .default = 1),
-    quantile = 0,
-    value = 0,
-    actual_value = 1,
-    params = list(dots),
-  )
-}
+# Debug mode will replace all forecasters with a fast dummy forecaster. Helps
+# with prototyping the pipeline.
 debug <- FALSE
```

Comment:
probably worth having a debug mode that also allows for the original way, in case it's a forecaster<->pipeline problem

Reply:
you mean one where we jump into browser / dump data on error in the slide?

Reply:
Yeah. a frequent pattern for me is to throw a `browser()` into a forecaster to see what it's doing step by step.

Reply:
cool, i brought it back